docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '2.3'
 
 services:
   pd0:
-    image: pingcap/pd:latest
+    image: pingcap/pd:v1.0.1
```

Comment:
Why is this?

Reply:
Just for testing lower version of TiDB, will change this later.

Reply:
DONE